src/main/kotlin/org/lsst/dax/albuquery/Results.kt

```diff
@@ -71,6 +71,7 @@ data class ParsedTable(
 data class ColumnMetadata(
     val name: String,
     val datatype: String?,
+    val description: String? = "",
```

Comment:
If the change is made from the comment above, this can actually be:

`val description: String,`

because it's never null at that point

Reply:
Ok.